src/main/java/com/evpa/GuessingGame.java

```diff
@@ -0,0 +1,76 @@
+package com.evpa;
+
+import java.util.Scanner;
+
+public class GuessingGame {
+    public static void main(String[] args) {
+
+        int computerNumber = (int) (Math.random()*100 + 1);
+        System.out.println("A random number has been generated...");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a guess between 1 and 100");
+        int userAnswer = scanner.nextInt();
+
+        // Guess 1
+        if (userAnswer <=0 || userAnswer >100) {
+            System.out.println("Invalid response");
+        }
+        else if (userAnswer == computerNumber ){
+            System.out.println("Correct! You win! See you next time!");
+            System.exit(0);
+        }
+        else if (userAnswer > computerNumber) {
+            System.out.println("Your guess is too high, guess again.");
+        }
+        else if (userAnswer < computerNumber) {
+            System.out.println("Your guess is too low, guess again.");
+        }
+        else {
+            System.out.println("Your guess is incorrect");
+        }
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 22:30, 43:51, 64:72.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.